docs/organizations/using-a-coding-standard.md

```diff
@@ -0,0 +1,94 @@
+# Using a coding standard
+
+Create a coding standard on your organization to define and apply global tool and code pattern configurations consistently across your repositories. You can also apply the coding standard to new repositories automatically by defining the coding standard as default.
+
+Use the organization coding standard to ensure that Codacy analyzes all your repositories with the same tool and code pattern settings. For example, configure the organization coding standard to ensure that all your repositories and teams are following the same global security rules or coding conventions.
+
+If you then customize the tool and code patterns for a specific repository, Codacy warns you that the repository will stop following the coding standard and ask for your confirmation.
+
+## Creating a coding standard {: id="creating"}
+
+<!--TODO
+
+-   Isn't it possible to configure the coding standard as the default while creating it? https://www.figma.com/file/Od7PrP0rpI2vvycq0iOvzI?node-id=904:2732#106911296
+
+-->
+
+Organizations on Codacy can have one coding standard. To create a coding standard for your organization:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Enter the name of the coding standard and click **Create standard**.
+
+    Optionally, select a repository that Codacy will use as a baseline to bootstrap the tool and pattern configurations for the new coding standard. This is useful if you already have a repository with code patterns configured and that you wish to use as a template for the coding standard.
+
+    ![Creating a new coding standard](images/coding-standard-create.png)
```

Comment:
@prcr Can we replace the screenshots when we have a staging available to do it?

Reply:
Yes, as soon as the new feature is available for testing I intend to update all screenshots with the final versions and also make any adjustments to the text that may be necessary. :+1: